zulipterminal/core.py

```diff
@@ -39,10 +39,7 @@ def __init__(self, config_file: str, theme_name: str,
         self.client = zulip.Client(config_file=config_file,
                                    client='ZulipTerminal/{} {}'.
                                           format(ZT_VERSION, platform()))
-        self.model = Model(self)
-        self.view = View(self)
-        # Start polling for events after view is rendered.
-        self.model.poll_for_events()
+        self.init_model_view()
```

Comment:
Currently `self.view` should exist before `initialize_loop` (which you just introduced), but this could cause self.view to not be set at this point.

I've not reviewed all the commits, but do we really need this to be asynch?

Reply:
We don't need `self.view` to initialize_loop :) `initialize_loop` loads `LoadingView` first and then when we have `self.view` ready, we display that. Also, In the latter commits I have split the function into two parts, first I load model, then initialize loop and then load the view.